webextensions/manifest/permissions.json

```
@@ -1049,12 +1049,34 @@
             }
           }
         },
+        "webRequestAuthProvider": {
+          "__compat": {
+            "description": "<code>webRequestAuthProvider</code>",
+            "support": {
+              "chrome": {
+                "version_added": "108"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false,
+                "notes": "Support planned, see <a href='https://bugzil.la/1820569'>bug 1820569</a>."
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
         "webRequestBlocking": {
           "__compat": {
             "description": "<code>webRequestBlocking</code>",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": true,
+                "notes": "In Manifest V3, no longer available for most extensions (the exception being policy-installed extensions). However, as of Chrome 108, can be used with \"webRequestAuthProvider\" permissions to ehable the supply of credentials asynchronously."
```

```suggestion
                "notes": "In Manifest V3, no longer available for most extensions (the exception being policy-installed extensions)."
```

webRequestBlocking cannot be used in Chrome's MV3. The webRequestAuthProvider permission is the alternative, but is independent of webRequestBlocking.